src/util/segmentation/operations/eraseIfSegmentIndex.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * EraseIfSegmentIndex - if the pixelData for the given pixelIndex
+ * is equal to the segmentIndesx, erase it (set it to 0).
```

Comment:
`segmentIndesx` typo

Reply:
Should probably `export default function EraseIfSegmentIndex(` or add `@name` / `@function` JSDoc